editor/src/core/model/element-metadata-utils.ts

```diff
@@ -807,7 +807,11 @@ export const MetadataUtils = {
     } => {
       // Note: This will not necessarily be representative of the structured ordering in
       // the code that produced these elements.
```

Comment:
Is this note still valid?

Reply:
Yes, because `reorderTree` is quite conservative, so things like blocks of generated elements will potentially still be out of order.